build.sbt

```diff
@@ -49,6 +49,7 @@ lazy val ws = (project in file("acdc-ws")).
       kineticpulse
     ),
     dependencyOverrides ++= Seq(
+      "com.google.guava" % "guava" % "32.0.0-jre",
```

Comment:
what's the guava dependency it suppose to have?

Reply:
they also generally does not do well with dependencies, we might want to test that too

Reply:
@realstraw sure, let me test it